addons/print/wizard/print_order_sendnow_wizard.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class print_order_sendnow_wizard(models.TransientModel):
+
+    _name = 'print.order.sendnow.wizard'
```

Comment:
Maybe removing the 'wizard' suffix could shorten the writing here. Otherwise the code is not easy to read (same for the field name herebelow).